net/tsdial/dnsmap_test.go

```diff
@@ -12,6 +12,14 @@ import (
 	"tailscale.com/types/netmap"
 )
 
+func nodeViews(v []*tailcfg.Node) []tailcfg.NodeView {
+	nv := make([]tailcfg.NodeView, len(v))
```

Comment:
Could also do
views.SliceOfViews(v).AsSlice()

Reply:
Considering it's repeated a few times

Reply:
I'm too dumb for that.

I can't follow these docs:

```go
// SliceOfViews returns a ViewSlice for x.
func SliceOfViews[T ViewCloner[T, V], V StructView[T]](x []T) SliceView[T, V] {
	return SliceView[T, V]{x}
}

// SliceView is a read-only wrapper around a struct which should only be exposed
// as a View.
type SliceView[T ViewCloner[T, V], V StructView[T]] struct {
```

"a struct which should only be exposed as a View".

I have no structs here?